test/OAuth2/TokenType/BearerTest.php

```diff
@@ -35,6 +35,20 @@ public function testInvalidContentType()
         $this->assertEquals($response->getParameter('error_description'), 'The content type for POST requests must be "application/x-www-form-urlencoded"');
     }
 
+    public function testMissingContentType()
+    {
+        $bearer = new Bearer();
+        $request = TestRequest::createPost(array(
+            'access_token' => 'ThisIsMyAccessToken'
+        ));
+
+        $param = $bearer->getAccessTokenParameter($request, $response = new Response());
+        $this->assertNull($param);
+        $this->assertEquals($response->getStatusCode(), 400);
+        $this->assertEquals($response->getParameter('error'), 'invalid_request');
+        $this->assertEquals($response->getParameter('error_description'), 'The content type for POST requests must be "application/x-www-form-urlencoded"');
```

Comment:
Non-blocker, but left is expected, right is actual, IIRC 🤔

Reply:
In my own defense I copied it from another test case.